Consistent -> Thorough

cwilso opened the pull request:
Fixes #117


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/AB-public/pull/151.html" title="Last updated on Jan 27, 2024, 5:33 PM UTC (7138758)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/AB-public/151/0d07246...7138758.html" title="Last updated on Jan 27, 2024, 5:33 PM UTC (7138758)">Diff</a>

tantek reviewed (approved):
This looks good to me, except I suggest also a minor grammar fix, "consistent horizontal reviews" (plural) since it's followed by a list, and in practice they are multiple horizontal reviews.

cwilso commented:
It was actually intentional to denote thorough review as a singular category, rather than plural to denote each current horizontal review.  Although it's followed by an illustrative list, it wasn't intended to be a closed-ended list (for example, the W3C Community might consider adding new aspects to that list (like we just added sustainability, despite not yet having an in-place horizontal review process for it).

Noting that, I changed the text, ptal.
